config/crds/apis.kcp.dev_apiexports.yaml

```diff
@@ -0,0 +1,79 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: apiexports.apis.kcp.dev
+spec:
+  group: apis.kcp.dev
+  names:
+    categories:
+    - kcp
+    kind: APIExport
+    listKind: APIExportList
+    plural: apiexports
+    singular: apiexport
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: "APIExport registers an API and implementation to allow consumption
+          by others through APIBindings. \n APIExports cannot be deleted until status.resourceSchemasInUse
+          is empty."
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: Spec holds the desired state.
+            properties:
+              latestResourceSchemas:
+                description: "latestResourceSchemas records the latest APIResourceSchemas
+                  that are exposed with this APIExport. \n The schemas can be changed
+                  in the life-cycle of the APIExport. These changes have no effect
+                  on existing APIBindings, but only on newly bound ones. \n For updating
+                  existing APIBindings, use an APIDeployment keeping bound workspaces
+                  up-to-date."
+                items:
+                  type: string
+                type: array
+                x-kubernetes-list-type: set
+            type: object
+          status:
```

Comment:
I could be wrong but I thought when discussing this with @ncdc there was some mention of exposing some kind of URL that points to a virtual workspace the controller can then watch in order to receive events from any workspace that imports the API

Reply:
We haven't gotten that far yet 😄